config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.10.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
Bump this to 0.11 https://github.com/brancz/kube-rbac-proxy/releases/tag/v0.11.0

Reply:
The latest available image is https://console.cloud.google.com/gcr/images/kubebuilder/global/kube-rbac-proxy v0.8.0